plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/DefaultedPipelineOptionsComponentTest.java

```diff
@@ -116,7 +115,7 @@ public void testDefaults_nulls() {
     assertTrue(component.isUseDefaultOptions());
     Map<String, String> values = component.getValues();
     // all empty/null settings should be turned to empty strings
```

Comment:
Comment doesn't fit with the line below it anymore. Should `testDefaults_nulls` be providing an email?

Reply:
`values.get(DataflowPreferences.ACCOUNT_EMAIL_PROPERTY)` would return the email associated with ADC regardless of the pipeline options being default or not. I will modify this comment to indicate it